lib/__tests__/click.test.ts

```diff
@@ -5,66 +5,109 @@ const credentials = {
   appId: "test"
 };
 
-let analyticsInstance;
```

Comment:
A lot of noise in click / conversion / view tests, but basically it amounts to adding a test case for additional parameters which required a bit of moving stuff around to reuse instances and test payloads.